org.mixedrealitytoolkit.input/Simulation/InputSimulator.cs

```diff
@@ -453,136 +453,138 @@ private void UpdateSimulatedController(Handedness handedness)
 
-                // Buttons available to hands and controllers
-                // Should we pass the trigger button straight through to the device?
-                // This will not smooth the trigger press; typically, we should
-                // modulate the trigger axis control ourselves for smooth pinch/unpinch,
-                // and this is false.
-                if (shouldUseTriggerButton)
-                {
-                    controls.TriggerButton = ctrlSettings.TriggerButton.action.IsPressed();
-                }
-                controls.GripButton = ctrlSettings.GripButton.action.IsPressed();
+                    // Buttons available to hands and controllers
+                    // Should we pass the trigger button straight through to the device?
+                    // This will not smooth the trigger press; typically, we should
+                    // modulate the trigger axis control ourselves for smooth pinch/unpinch,
+                    // and this is false.
+                    if (shouldUseTriggerButton)
+                    {
+                        controls.TriggerButton = ctrlSettings.TriggerButton.action.IsPressed();
+                    }
+                    controls.GripButton = ctrlSettings.GripButton.action.IsPressed();
 
-                if (ctrlSettings.SimulationMode == ControllerSimulationMode.MotionController)
-                {
-                    // Axes available to controllers
-                    // TODO: "soon"
-                    // controls.Primary2DAxis = ctrlSettings.Primary2DAxis.action.ReadValue(float)();
-                    // controls.Secondary2DAxis = ctrlSettings.Secondary2DAxis.action.ReadValue(float)();
-
-                    // Buttons available to controllers
-                    // TODO: "soon"
-                    // controls.MenuButton = ctrlSettings.MenuButton.action.ReadValue(float)() > 0f;
-                    // controls.PrimaryButton = ctrlSettings.PrimaryButton.action.ReadValue(float)() > 0f;
-                    // controls.SecondaryButton = ctrlSettings.SecondaryButton.action.ReadValue(float)() > 0f;
-                    // controls.PrimaryTouch = ctrlSettings.PrimaryTouch.action.ReadValue(float)() > 0f;
-                    // controls.SecondaryTouch = ctrlSettings.SecondaryTouch.action.ReadValue(float)() > 0f;
-                    // controls.Primary2DAxisClick = ctrlSettings.Primary2DAxisClick.action.ReadValue(float)() > 0f;
-                    // controls.Secondary2DAxisClick = ctrlSettings.Secondary2DAxisClick.action.ReadValue(float)() > 0f;
-                    // controls.Primary2DAxisTouch = ctrlSettings.Primary2DAxisTouch.action.ReadValue(float)() > 0f;
-                    // controls.Secondary2DAxisTouch = ctrlSettings.Secondary2DAxisTouch.action.ReadValue(float)() > 0f;
+                    if (ctrlSettings.SimulationMode == ControllerSimulationMode.MotionController)
+                    {
+                        // Axes available to controllers
```

Comment:
What's our opinion on keeping in commented out TODO code?

Reply:
Good question! I believe these were left in by @david-c-kline. Dave, do you expect these TODOs to be resolved soonish or should I refactor them out into an issue?

Reply:
Not sure how soon these are going to be addressed. Filing an issue and referencing in the todo comment feels right. Lets others see what needs to be done and maybe give them an easy first issue.

Reply:
Filed https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/issues/734 to track this